robot_giua_ki:
  # Định nghĩa position controllers cho các khớp
  part_1_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: part_1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  part_2_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: part_2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  part_3_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: part_3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
